Description: fix DEB_BUILD_OPTIONS=nostrip
Author: Stefan Fritsch <sf@debian.org>
Last-Update: 2011-11-01

--- a/Makefile
+++ b/Makefile
@@ -33,7 +33,7 @@
 export PERL_BADLANG
 
 CXXX=$(CXD_assert)
-LDALL=$(LDXX) -s $(LDFLAGS) $(LIBS)
+LDALL=$(LDXX) $(LDFLAGS) $(LIBS)
 ifeq ($(ENABLE_DEBUG), no)
 CXXX=$(CXD_no)
 endif
@@ -101,8 +101,8 @@
 IDE_TARGETS_debug := $(patsubst %,%.yes, $(TARGETS))
 IDE_TARGETS_release := $(TARGETS)
 
-CXD_assert =$(CXX) -s -O2
-CXD_no     =$(CXX) -s -DNDEBUG -O3
+CXD_assert =$(CXX) -g -O2
+CXD_no     =$(CXX) -DNDEBUG -O3
 # CXD_yes    =$(CXX) $(GFLAG) -lefence
 CXD_yes    =$(CXX) $(GFLAG)
 CXD_checker=checkerg++ $(GFLAG)
